python/shotgun_model/util.py

```diff
@@ -148,6 +148,12 @@ def sanitize_qt(val):
             new_val[safe_key] = safe_val
         return new_val
 
+    elif isinstance(val, long):
+        try:
+            val = int(val)
```

Comment:
I just tried, and `int(1000000000000000000000000000000000000000000000000)` will return a long, so no need to try/except

Reply:
Thanks I just removed try - except